### PR TITLE
Allows Send JUP Widget To Send Messages

### LINKED ISSUE #67: Add Ability to Include a Message in Transactions
Since sending a message with transactions is an essential activity to send mainnet JUP to exchanges, this is a very high priority issue and must be done before delivery.

### PR DESCRIPTION
**Pull request by outragedhuman**
Closes #67

### CODE REVIEW

**Review by outragedhuman — COMMENTED** [review]
looks good but it would be worth considering moving the checkbox to allow message sending to the seed collection window itself, which would allow more flexibility in where messages can be sent from without needing to add it in many places

Issue created: https://github.com/Up-North-Coding/jovian/issues/104

File: src/contexts/APIRouterContext/Provider.tsx [hunk 1]
```diff
@@ -250,7 +267,7 @@ const APIRouterProvider: React.FC = ({ children }) => {
       }
 
       // TODO: should consider doing something with the return of this function call
-      await afterSecretCB.current(userSecretInput);
+      await afterSecretCB.current(userSecretInput, messageInput);
```
Comment (vwalsh) [inline src/contexts/APIRouterContext/Provider.tsx:270]: just to be clear, message input is being ignored by the other _handle functions.

Comment (vwalsh) [reply src/contexts/APIRouterContext/Provider.tsx:270]: perhaps a better way to do it is to check for a message to exist? i need to talk it over and think on it a bit more

**Review by vwalsh — COMMENTED** [review]
reviewed, some discussion to have

File: src/contexts/APIRouterContext/Provider.tsx (hunk 1 above)
Comment (outragedhuman) [reply src/contexts/APIRouterContext/Provider.tsx:270]: yea I'm aware it's being ignored in the others, i didn't really like the approach I had to take but I didn't see a better way to stitch in additional fields to consume

Comment (vwalsh) [reply src/contexts/APIRouterContext/Provider.tsx:270]: see issue #115

**Pull request merged**